src/main/java/games/strategy/engine/data/RepairFrontier.java

```diff
@@ -22,22 +22,14 @@ public RepairFrontier(final String name, final GameData data) {
     super(name, data);
   }
 
-  public void addRule(final RepairRule rule) {
+  void addRule(final RepairRule rule) {
     if (m_rules.contains(rule)) {
       throw new IllegalStateException("Rule already added:" + rule);
     }
     m_rules.add(rule);
     m_cachedRules = null;
   }
 
-  public void removeRule(final RepairRule rule) {
```

Comment:
I couldn't find any reflection references using similar search parameters as above.

Searching through the history, all calls I can find to `removeRule()` resolve to `ProductionFrontier#removeRule()`.  Specifically via the `RemoveProductionRule` change.  I cannot find an equivalent `RemoveRepairRule` in the history.  So I'm guessing `RepairFrontier#removeRule()` was added simply for a perceived future requirement.